README.md

```diff
@@ -329,6 +329,13 @@ You can create a `.nvmrc` file containing version number in the project root dir
 
 For example, to make nvm default to the latest 5.9 release for the current directory:
 
+.nvmrc file:
```

Comment:
There's many ways to create a file and add content to it; it's critical that there be no trailing whitespace, including a trailing newline; the command allows it to be precisely conveyed, with zero possibility of error. You don't need to know what the command does, you just need to run it.